docs/cloud/dev/spryker-cloud-commerce-os/add-variables-in-the-parameter-store.md

```diff
@@ -58,6 +58,12 @@ Path examples:
 
 * /deans_jeans/prod/config/app/public/mail_host
 
+### Reserved variables
+Reserved variables are variable names that have special meaning in Spryker Cloud Commerce OS. Since their names are reserved, you cannot define your own variables using these names.
+The reserved variable names are the following:
+* `DUMMY_INIT`
+* `SPRYKER_*`
+
```

Comment:
Could you wrap this into the warning block please?

Reply:
Done. I put content in a warning block.